modules/etcpak/config.py

```diff
@@ -1,5 +1,5 @@
 def can_build(env, platform):
-    return env.editor_build
+    return True
```

Comment:
We should measure the size impact on a `template_release` build to make sure that the increase in binary size is worth the added feature.